src/utils/list_storage_entries.rs

```diff
@@ -69,6 +69,7 @@ pub fn list_storage_entries_any(
         RuntimeMetadata::V13(_opaque) => Box::new(core::iter::empty()),
         RuntimeMetadata::V14(m) => Box::new(m.storage_entries_list()),
         RuntimeMetadata::V15(m) => Box::new(m.storage_entries_list()),
+        RuntimeMetadata::V16(_opaque) => Box::new(core::iter::empty()),
```

Comment:
Should we also extract entries for the unstable feature flag? Would this break anything?

Reply:
It might break things in that the unstable metadata can change without a semver break in `frame-metadata` I guess? In practise it shouldn't be an issue becasue we're not changing anything there though!